docs/content/en/components/k8ssandra-operator/_index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "K8ssandra Operator"
+linkTitle: "K8ssandra Operator"
+weight: 1
+description: An introduction to K8ssandra Operator, our newest implementation.
+---
+
+K8ssandra Operator provides multi-cluster and multi-region support for Apache Cassandra&reg; databases deployed in Kubernetes environments. K8ssandra operator also works in single-cluster and single-region Kubernetes deployments.
+
+You can [install](https://docs-staging-v2.k8ssandra.io/install/local/) K8ssandra Operator in:
+
+* A local Kubernetes 1.19+ environment, such as kind, K3D, or minikube
+* A supported cloud provider's Kubernetes environment
+
+## Architecture
+
+K8ssandra Operator consists of two primary components:
+
+* A control plane
+* A data plane
+
+The control plane creates and manages objects that exist only in the API server. The control plane does not deploy or manage pods. Note that the control plane can be installed in only one cluster; that is, in the control plane cluster.
```

Comment:
This isn't strictly true in that a control-plane cluster *can* be used to deploy a datacenter, we'd just recommend for folks not to.  So I'm not totally sure how you want to word this, but we'll want to remove the bit about `The control plane does not deploy or manage pods`

Reply:
Ok thanks will fix that and will grab John Sanda's deep dive architecture info.